docs/LV_client/sensors/mono_fisheye_camera_poly1.md

```diff
@@ -1,12 +1,12 @@
-# mono_fisheye_camera.vi
+# mono_fisheye_camera_poly1.vi
 
 <p class="img_container">
-<img class="lg_img" src="../mono_fisheye_camera.png"/>
+<img class="lg_img" src="../mono_fisheye_camera_poly1.png"/>
 </p>
 
 ### Description
 
-Configures and reads the data stream for a Fisheye Camera sensor and outputs a picture with the specified format in the stream dimensions.
+Configures and reads the data stream for a Fisheye Camera, this camera is a scaramuzza model based. 
```

Comment:
This sentence flows better with based prior to scaramuzza. "Configures and reads the data stream for a Fisheye Camera, this camera is based on a Scaramuzza model." or "Configures and reads the data stream for a Fisheye Camera, this camera is based on Scaramuzza's model." I think Scaramuzza should be capitalized.